docs/api/rest_api/rest_api_reference/input/ibexa-content-types.raml

```diff
@@ -39,6 +39,41 @@ get:
             description: OK - returns a list of content types.
         401:
             description: Error - The user has no permission to read the content types.
+/view:
+    post:
+        displayName: Find content types
+        description: Executes a query and returns a View that includes the results. Refer to [Search Criteria Reference](/en/latest/search/content_type_search_reference/content_type_criteria/).
+        headers:
+            Accept:
+                description: The ContentTypeList schema is returned in XML or JSON format.
+                example: |
+                    application/vnd.ibexa.api.ContentTypeList+xml
```

Comment:
The example from https://github.com/ibexa/rest/pull/196 mentions:
` Accept => application/vnd.ibexa.api.ContentTypeView+json`

here you have `application/vnd.ibexa.api.ContentTypeList`, which one is correct?